### ISSUE TITLE
default: PHP Fatal error

### ISSUE THREAD

**Issue by 121onto** [directus/directus-vagrant#12]
Hi,

I tried installing this and I get the following error when I run `vagrant up`:

```bash
==> default: PHP Fatal error:  Cannot pass parameter 2 by reference in /var/www/html/directus/api/core/Directus/Util/Installation/Console.php on line 178
```

The server is on, I can open http://192.168.33.6/login.php in my browser and I get a login screen.  However, the default login credentials don't work.

**Comment by WellingGuzman** [comment]
Hi @121onto I've fixed the issue here https://github.com/directus/directus/commit/26801855580d9bea757d9650cbc11682fd83e52b

Try reinstalling Directus again, and let me know if this works for you.

Thanks, for letting us know.

**Comment by 121onto** [comment]
Wow, that's what I call responsive.  Thank  you!  

It installs now but I can't login... could it be a permissions issue?

**Comment by WellingGuzman** [comment]
We are here to help :smiley: 

Are you using the default email/password `admin@admin.com/admin`?

If so, can you check if the user record was inserted into the `directus_users` table?

Here is a way you can see the `directus_users` table records.

On directus-vagrant directory log into the virtual machine:
```shell
$ vagrant ssh
```

Then once inside the virtual machine, connect to mysql:

```shell
$ mysql -u root -p123 directus
```

Once connected to mysql run this query (on the console)

```shell
$ select * from directus_users;
```

Let me know if there's a user with `admin@admin.com` email.

**Comment by 121onto** [comment]
Yes, the user is there:

```
mysql> select * from directus_users;
+----+--------+------------+-----------+-----------------+------------------------------------------+---------------+-------+-------------+------------------+----------+----------------+------------+-------------+-----------+------+-------+--------+----------------+----------+-------+---------+------+-------+------+
| id | active | first_name | last_name | email           | password                                 | salt          | token | reset_token | reset_expiration | position | email_messages | last_login | last_access | last_page | ip   | group | avatar | avatar_file_id | location | phone | address | city | state | zip  |
+----+--------+------------+-----------+-----------------+------------------------------------------+---------------+-------+-------------+------------------+----------+----------------+------------+-------------+-----------+------+-------+--------+----------------+----------+-------+---------+------+-------+------+
|  1 |      1 | Admin      | User      | admin@admin.com | ed18e76b6fe9b68b666017115fa2467d168c0d5b | 5697fae83e118 |       |             | NULL             |          |              1 | NULL       | NULL        |           |      |     1 | NULL   |           NULL | NULL     | NULL  | NULL    | NULL | NULL  | NULL |
+----+--------+------------+-----------+-----------------+------------------------------------------+---------------+-------+-------------+------------------+----------+----------------+------------+-------------+-----------+------+-------+--------+----------------+----------+-------+---------+------+-------+------+
1 row in set (0.00 sec)
```

**Comment by 121onto** [comment]
Also, yes, I am using the default email/password of admin@admin.com/admin.

**Comment by WellingGuzman** [comment]
I'm looking at the issue right now, I've installed couple of times right now and I can log in.

can you delete the directory `directus` inside `directus-vagrant/html` (*IMPORTANT* if you don't have any important files in here) and delete the database and re-run vagrant.

```shell
# To delete the database logged into mysql
$ drop database directus; create database directus;
```

Reload vagrant with provisioning
```shell
$ vagrant reload --provision
```

If that doesn't work I'll see what's wrong with Directus.

**Comment by 121onto** [comment]
It was a problem with virtualbox.  I believe the problem occurred because I deleted the `directus-vagrant` directory without running `vagrant destroy`.  I don't know exactly what I did to fixed it, but here are the steps I took:

1. uninstall virtualbox and vagrant
2. deleted `~/VirtualBox VMs` 
3. reinstalled virtualbox and vagrant
4. stopped all running vms, see http://stackoverflow.com/questions/15408969
5. opened virtualbox GUI, clicked on each vm instance and ctrl+R to remove them
6. removed and reinstalled `directus-vagrant`

Thanks again for your help!
